core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -1050,7 +1050,11 @@ private[spark] class Executor(
     // Bootstrap the list of jars with the user class path.
     val now = System.currentTimeMillis()
     userClassPath.foreach { url =>
-      currentJars(url.getPath().split("/").last) = now
```

Comment:
Hm, did you set `spark.executor.extraClassPath` as `/`? This is user-specified class path.

Reply:
We mistakenly set `spark.executor.extraClassPath` to an empty string, which triggered this bug.

Reply:
This isn't related to starting at the root or not. Setting `spark.executor.extraClassPath` matters.

Reply:
You are right, there is something wrong with my previous description. However, when `spark.executor.extraClassPath` is set to an empty string and the current directory is the root directory, an exception will occur because Spark does not verify whether spark.executor.extraClassPath is empty.

Reply:
Let's probably focus on fixing the error message instead of making sliently igoring